### PR TITLE
Pip

### PR DESCRIPTION
**Pull request by ElizaSapir**

### CODE REVIEW

**Comment by itaykinnrot** [comment]
@ElizaSapir 2 remarks:
1. remove this file -  modules/KalturaSupport/KalturaSupport.php
2. add verification that we're on iPad/iOS9+ in the embedPlayer

**Comment by ElizaSapir** [comment]
@itaykinnrot:
1.  Done - KalturaSupport.php was removed
2. Already resolved - https://github.com/kaltura/mwEmbed/commit/b5730f8605294fcbf73ff8fadfcf41b6ca2bcc8e#diff-889974041dfb89d1ace71105e72d8cd2R27 (we have this condition on isSafeEnvironment)

**Comment by amirch1** [comment]
@itaykinnrot - do you want to add a test page?

File: modules/KalturaSupport/components/pipBtn.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+				'parent': 'controlsContainer',
+				'accessibleControls': false,
+				'order': 5,
+				'showTooltip': true,
+				'align': "right",
+				"displayImportance": "medium"
+			},
+
+			isDisabled: true,
+			pipTitle: gM( 'mwe-embedplayer-pip' ),
```
Comment (amirch1) [inline modules/KalturaSupport/components/pipBtn.js]: @ElizaSapir : mwe-embedplayer-pip must be defined in the localization file

File: modules/KalturaSupport/components/pipBtn.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+			},
+			isSafeEnviornment: function(){
+				return mw.isIOS9() && mw.isIpad() && !mw.isIpad2() ;
+			},
+			getComponent: function() {
+				var _this = this;
+				if( !this.$el ) {
+					this.$el  = $( '<button />' )
+						.attr( 'title', this.nextTitle )
+						.addClass( "btn btnNarrow icon-new-tab pull-right" )
```
Comment (amirch1) [inline modules/KalturaSupport/components/pipBtn.js]: @ElizaSapir - please remove the btnNarrow class - it causes the button to look bad and is not needed here

File: modules/MwEmbedSupport/mediawiki/mediawiki.client.js [hunk 3]
```diff
@@ -54,6 +54,9 @@
 	mw.isIpad = function () {
 		return ( userAgent.indexOf('iPad') != -1 );
 	};
+	mw.isIpad2 = function () {
+		return ( mw.isIpad() && ( window.devicePixelRatio && window.devicePixelRatio < 2 ) );
```
Comment (amirch1) [inline modules/MwEmbedSupport/mediawiki/mediawiki.client.js]: @ElizaSapir - no need for the additional brackets here

File: modules/KalturaSupport/components/pipBtn.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+				});
+			},
+			isSafeEnviornment: function(){
+				return mw.isIOS9() && mw.isIpad() && !mw.isIpad2() ;
+			},
+			getComponent: function() {
+				var _this = this;
+				if( !this.$el ) {
+					this.$el  = $( '<button />' )
+						.attr( 'title', this.nextTitle )
```
Comment (amirch1) [inline modules/KalturaSupport/components/pipBtn.js]: @ElizaSapir - shouldn't the title be this.pipTitle ?

**Pull request merged**